[2024-06-03T06:52:44.950+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-03T06:52:44.990+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag.load manual__2024-06-03T06:52:13.043733+00:00 [queued]>
[2024-06-03T06:52:45.003+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag.load manual__2024-06-03T06:52:13.043733+00:00 [queued]>
[2024-06-03T06:52:45.005+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-03T06:52:45.026+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load> on 2024-06-03 06:52:13.043733+00:00
[2024-06-03T06:52:45.043+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=6428) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-03T06:52:45.045+0000] {standard_task_runner.py:63} INFO - Started process 6430 to run task
[2024-06-03T06:52:45.043+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_dag', 'load', 'manual__2024-06-03T06:52:13.043733+00:00', '--job-id', '308', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmp1qvaumcn']
[2024-06-03T06:52:45.048+0000] {standard_task_runner.py:91} INFO - Job 308: Subtask load
[2024-06-03T06:52:45.077+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-06-03T06:52:45.129+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_dag.load manual__2024-06-03T06:52:13.043733+00:00 [running]> on host aa989158b36e
[2024-06-03T06:52:45.241+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_dag' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2024-06-03T06:52:13.043733+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-03T06:52:13.043733+00:00'
[2024-06-03T06:52:45.243+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-03T06:52:45.298+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/google/cloud/firestore_v1/base_collection.py:302 UserWarning: Detected filter using positional arguments. Prefer using the 'filter' keyword argument instead.
[2024-06-03T06:52:52.257+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-03T06:52:52.258+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-03T06:52:52.270+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_dag, task_id=load, run_id=manual__2024-06-03T06:52:13.043733+00:00, execution_date=20240603T065213, start_date=20240603T065244, end_date=20240603T065252
[2024-06-03T06:52:52.299+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-03T06:52:52.324+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-03T06:52:52.327+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
